build/BUILD.bazel

```diff
@@ -2,7 +2,6 @@ package(default_visibility = ["//visibility:public"])
 
 load("@io_k8s_repo_infra//defs:build.bzl", "release_filegroup")
 load(":container.bzl", "multi_arch_container", "multi_arch_container_push")
-load(":platforms.bzl", "SERVER_PLATFORMS", "UBI_PLATFORMS", "for_platforms")
```

Comment:
need to keep `SERVER_PLATFORMS` and `for_platforms`

Reply:
ooops...